backend/tests/helpers/cleanups.py

```diff
@@ -21,25 +21,31 @@ def es_cleanup(es):
     After upgrading to invenio 1.2 we will be able
     to remove only specified index
     """
-    from invenio_search import current_search, current_search_client
+    from invenio_search import current_search
 
     es.indices.refresh()
-    existing_mappings = set(es.indices.stats()["indices"].keys())
+    existing_mappings = []
+    [
+        existing_mappings.extend(x["aliases"].keys())
+        for x in es.indices.get_alias("*").values()
+    ]
+    existing_mappings = set(existing_mappings)
```

Comment:
using comprehensions for side effects is not nice. I would use a for loop, or better, if you want to keep using a comprehension
```python
from itertools import chain
existing_mappings_nested = (x["aliases"].keys() for x in es.indises.get_alias("*").values())
existing_mappings = set(chain.from_iterable(existing_mappings_nested))
```